### PR TITLE
File reformats

### PR DESCRIPTION
**Pull request by MaFeLP**
Started reformatting the following files to enforce PEP8 and help #44:

- [X] pokete_data/
  - [X] __init__.py
  - [X] attacks.py
  - [X] effects.py
  - [X] map_data.py
  - [X] npcs.py
  - [X] poketes.py
  - [X] trainers.py
- [X] pokete_classes/
  - [X] classes.py
  - [X] ui_elements.py
- [X] gen-wiki.py
- [X] pokete.py

### CODE REVIEW

File: pokete.py [hunk 1]
```diff
@@ -404,18 +421,22 @@ def attack(self, attac, enem):
...
             self.oldhp = self.hp
             effectivity = (1.3 if enem.type.name in attac.type.effective
-                            else 0.5
-                            if enem.type.name in attac.type.ineffective
-                            else 1)
-            n_hp = round((self.atc * attac.factor / (enem.defense if enem.defense >= 1 else 1))*random.choices([0, 0.75, 1, 1.26], weights=[attac.miss_chance+self.miss_chance, 1, 1, 1], k=1)[0]*effectivity)
+                           else 0.5
+                           if enem.type.name in attac.type.ineffective
+                           else 1
+                           )
```
Comment (lxgr-linux) [inline pokete.py:427]: Why does the bracket close here and not the lone before?

File: pokete.py [hunk 2]
```diff
@@ -593,25 +617,27 @@ def set_args(self, si):
...
+                      for poke in si["pokes"]]
         for j, poke in enumerate(self.pokes):
             poke.set_ap(si["pokes"][j]["ap"])
             if "effects" in si["pokes"][j]:
                 for e in si["pokes"][j]["effects"]:
                     poke.effects.append(getattr(effects, e)(poke))
         try:
-            if si["map"] in ["centermap", "shopmap"]:  # Looking if figure would be in centermap, so the player may spawn out of the center
+            # Looking if figure would be in centermap, so the player may spawn out of the center
+            if si["map"] in ["centermap",
```
Comment (lxgr-linux) [inline pokete.py:637]: Why does the line wrap here.

File: pokete.py [hunk 3]
```diff
@@ -1101,26 +1145,27 @@ def add(self):
 
 class Buy:
     """Menu to buy items in, is triggered in shop"""
+
     def __init__(self):
-        self.box = ChooseBox(height-3, 35, "Shop")
-        self.box2 = Box(7, 21,)
+        self.box = ChooseBox(height - 3, 35, "Shop")
+        self.box2 = Box(7, 21, )
```
Comment (lxgr-linux) [inline pokete.py:1151]:
```suggestion
        self.box2 = Box(7, 21)
```
This comma can be removed.

File: pokete.py [hunk 4]
```diff
@@ -1101,26 +1145,27 @@ def add(self):
...
         self.items = [Inv.poketeball, Inv.superball, Inv.healing_potion,
-                    Inv.super_potion, Inv.ap_potion]
+                      Inv.super_potion, Inv.ap_potion]
         self.box.add_c_obs([se.Text(f"{obj.pretty_name} : {obj.price}$")
-                                for obj in self.items])
-        self.money_label = se.Text(str(figure.get_money())+"$")
+                            for obj in self.items])
+        self.money_label = se.Text(str(figure.get_money()) + "$")
```
Comment (lxgr-linux) [inline pokete.py:1156]:
```suggestion
        self.money_label = se.Text(f"{figure.get_money()}$")
```
An f-string may be suited better.

File: pokete.py [hunk 5]
```diff
@@ -1216,8 +1262,17 @@ def __call__(self):
 
 class About:
     """The about text, that can be triggered in the menu"""
+
     def __init__(self):
-        self.box = InfoBox(liner(f"Pokete v{VERSION} -- {CODENAME}\n by lxgr-linux <lxgr@protonmail.com>\n \n This software is licensed under the GPL3, you should have gotten a copy of the GPL3 license alongside this software.\n Feel free to contribute what ever you want to this game, new Pokete contributions are especially welcome.\n For this see the comments in the definations area.\n You can contribute here: https://github.com/lxgr-linux/pokete", 60, pre=""), map=movemap)
+        self.box = InfoBox(liner(f"""
```
Comment (lxgr-linux) [inline pokete.py:1267]: I guess there was a cause I didn't do that in the first place.

File: pokete.py [hunk 6]
```diff
@@ -1578,15 +1636,16 @@ def balls_label_rechar():
     """Rechars the balls label"""
     movemap.balls_label.text = ""
     for i in range(6):
-        movemap.balls_label.text += "-" if i >= len(figure.pokes) or figure.pokes[i].identifier == "__fallback__" else "o" if figure.pokes[i].hp > 0 else "x"
+        movemap.balls_label.text += "-" if i >= len(figure.pokes) or figure.pokes[
```
Comment (lxgr-linux) [inline pokete.py:1639]: This one looks cursed.

File: pokete.py [hunk 7]
```diff
@@ -2522,19 +2590,19 @@ def map_additions():
...
 ~~~~~~~~
 ~~~~~~
  ~~~~
- ~~""", esccode=Color.blue, ignore=Color.blue+" "+Color.reset,
-        ob_class=HightGrass,
-        ob_args={"pokes": ["karpi", "blub", "clampi"],
-                 "minlvl": 540, "maxlvl": 640},
-        state="float")
+ ~~""", esccode=Color.blue, ignore=Color.blue + " " + Color.reset,
+                          ob_class=HightGrass,
```
Comment (lxgr-linux) [inline pokete.py:2602]: Why is this indented.

File: pokete_data/map_data.py [hunk 8]
```diff
@@ -1,14 +1,14 @@
...
-#         "hard_obs": {
-#         },
-#         "soft_obs": {
-#         },
-#         "dors": {
-#         },
-#         "balls": {
-#         }
-#     },
+    #     "playmap_1": {
```
Comment (lxgr-linux) [inline pokete_data/map_data.py:2]: Why is this indented?

File: pokete_data/map_data.py [hunk 9]
```diff
@@ -1656,7 +1654,8 @@
             "tree_group_1": {
                 "x": 0,
                 "y": 0,
-                "txt": """()(()())))))((())(()))((((())))((((()))))))))(()())((()()))()))()()()())))(((())()))()())))(()((())))(())()())))()(()())
+                "txt": """
```
Comment (lxgr-linux) [inline pokete_data/map_data.py:1657]: This code produces a Coordinate error and a uneeded line at line of spaces at the start of the screen.

File: pokete_data/map_data.py [hunk 10]
```diff
@@ -1928,7 +1928,8 @@
             "tree_group_1": {
                 "x": 0,
                 "y": 0,
-                "txt": """(())(()()))(())()()()))((())())(()))()())()\___   \________       \______            \        \\
+                "txt": """
```
Comment (lxgr-linux) [inline pokete_data/map_data.py:1931]: This also produces a CoordinateError.

File: pokete_data/map_data.py [hunk 11]
```diff
@@ -2179,7 +2180,8 @@
             "tree_group_1": {
                 "x": 0,
                 "y": 0,
-                "txt":"""))())())())))()()())))((())(()()()))()))())(((()())())()()()()())))())))))(()(()()()()))((()  #    #  (((()))()((()()(()))((()()())((()()))()()))())))
+                "txt": """
```
Comment (lxgr-linux) [inline pokete_data/map_data.py:2183]: This also produces a CoordinateError.

File: pokete.py (hunk 1 above)
Comment (MaFeLP) [reply pokete.py:427]: Better readability, so you know by looking, when the nested if-else statements are done.

File: pokete.py (hunk 2 above)
Comment (MaFeLP) [reply pokete.py:637]: The comment would make the line longer than 120 characters, which raises a PEP8 warning E501

File: pokete.py (hunk 5 above)
Comment (MaFeLP) [reply pokete.py:1267]: Wow. So se bug?

File: pokete.py (hunk 6 above)
Comment (MaFeLP) [reply pokete.py]: Ok. See it.

Fixed in 900135ff7a571b957d878156345a9a4e82278b0f

File: pokete.py (hunk 7 above)
Comment (MaFeLP) [reply pokete.py:2602]: It is indented at the same level as the bracket in line 2593, so you can see that this code belongs to se.Text(...

File: pokete_data/map_data.py (hunk 9 above)
Comment (MaFeLP) [reply pokete_data/map_data.py]: Fixed in ee3504b7128cc58e2dcc0a451993ace913c5fc08

File: pokete_data/map_data.py (hunk 10 above)
Comment (MaFeLP) [reply pokete_data/map_data.py]: Fixed in ee3504b7128cc58e2dcc0a451993ace913c5fc08

File: pokete_data/map_data.py (hunk 11 above)
Comment (MaFeLP) [reply pokete_data/map_data.py]: Fixed in ee3504b7128cc58e2dcc0a451993ace913c5fc08

File: pokete_data/map_data.py (hunk 8 above)
Comment (MaFeLP) [reply pokete_data/map_data.py:2]: Comments should be indented the same width as their scope, in this case the following "real" playmap_1

File: pokete.py (hunk 1 above)
Comment (MaFeLP) [reply pokete.py:427]: But we could put it in the line before as well, if that makes you happy :)

File: pokete.py (hunk 5 above)
Comment (MaFeLP) [reply pokete.py]: Fixed in 0e3baa2ea0987955c57bd19a4e5fd05a1abaff4f

Comment (MaFeLP) [reply pokete.py]: Just use two spaces in a new line and everything should be fine.

File: pokete.py (hunk 2 above)
Comment (lxgr-linux) [reply pokete.py:637]: i see, why you sperated the comment, but not why `if si["map"] in ["centermap", "shopmap"]:` wraps.

**Review by lxgr-linux — APPROVED** [review]
Looks great!

**Pull request merged**